sw/device/tests/BUILD

```diff
@@ -1868,3 +1868,23 @@ opentitan_functest(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
+opentitan_functest(
+    name = "chip_power_sleep_load",
+    srcs = ["chip_power_sleep_load.c"],
+    deps = [
```

Comment:
You will need to add the line [`targets = ["dv"],`](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/tests/sim_dv/BUILD;drc=ef2bd52cdfb9e0e10c396a328df3c8c94bdd48e3;l=11) so that this test is only built for the `sim_dv` platform. Otherwise, this test will be run on the FPGA, and fail, and rejected by CI

Reply:
see my note above.